config/database.pg.yml

```diff
@@ -15,7 +15,7 @@ base: &base
   adapter: postgresql
   encoding: utf8
   username: root
-  pool: 1
+  pool: 5

```

Comment:
@imtayadeway Is this related to this PR?

I think the UI and web service worker should be already set to 5 as are other workers:

```
$ grep -B 1 connection_pool_size config/vmdb.tmpl.yml
    :replication_worker:
      :connection_pool_size: 5
--
    :smis_refresh_worker:
      :connection_pool_size: 5
--
    :ui_worker:
      :connection_pool_size: 5
--
    :web_service_worker:
      :connection_pool_size: 5
```

@matthewd Note:  the entire reason this is set to 1 is that activerecord + threads (back in rails 2.3/3.0/3.1 days I think) would establish database connections in the pool, reserve it for a thread, another thread did a db query before that thread returned it to the pool, a new connection would be established, both would go back to the pool eventually but never be disconnected.  In other words, activerecord would never timeout and disconnect an inactive connection so if you have several threads spin up and do db activity once per hour, you'd get a bunch of db connections in a single process that are used a handful and then many of them go unused for a long time.  

If you have many appliances talking to a single one with a database, you can easily have 10s of processes with several connections per process (from threads), leading to postgresql with easily more than 100+ connections,  of which many of those connections were completely unused.

Now, we don't typically do much with threads and the database since we went process over threads due to rails 2.3, gems we used, and our own code not being totally thread safe.

If Rails 4.2 times out inactive connections in the pool or we just want to re-evaluate this decision to see if it's still needed, I'm good with that... just want to explain why it's the way it is.